<?xml version="1.0"?>

<launch>
	<arg name="serial_comm" default="true"/>

	<rosparam file="$(find rover_20_control)/config/autonomous.yaml" command="load"/>

	<node pkg="rover_20_control" type="uberotonom20.py" name="autonomous_drive" output="screen"/>

	<group if="$(arg serial_comm)">
		<node pkg="rover_20_control" type="cmd_20.py" name="cmd_20_node"/>
		<node pkg="rover_19_simulator" type="odom_publisher.py" name="odom_pub_node"/>
	</group>

</launch>